Gemfile

```diff
@@ -15,3 +15,7 @@ if version == "master"
 else
   gem "rails", "~> #{version}.0"
 end
+
+group :test do
+  gem "timecop"
+end
```

Comment:
Ok, but in these tests we don't use ActiveRecord inherited classes in tests.
The updated_at method is created "by hand" : https://github.com/rails-api/active_model_serializers/blob/master/test%2Ffixtures%2Fporo.rb#L21

And, the problem here is that it assumes that all code is executed in the same second. That's why sometimes it fails.